examples/clean-state/README.md

```diff
@@ -0,0 +1,16 @@
+# Clean state contract in JavaScript
+
+This is an equivalent JavaScript implementation of the clean state example. Every user can send and receive a non-fungible token.
+
```

Comment:
I suggest to add a short instruction about how to use this contract. In summary, the steps are:
1. Deploy this contract to the account where you want to clean up state
2. Call the contract with known state keys to clean state. If delete all keys doesn't fit gas limit, you may need to do it a few times
3. Once finished clean up state, Deploy the original contract or delete the contract code with `near js remove`

Reply:
Done, PTAL